### ISSUE TITLE
Additional explanation when there are no custom speech personalities

### ISSUE THREAD

**Issue by richardbuckle** [EDCD/EDDI#237]
Users are frequently confused by the requirement to copy the default personality before editing. On the speech responder tab, if and only if the user has zero custom personalities, display an additional caption. Something like:

> The default personality is read-only. if you wish to make changes, create a copy.

**Comment by richardbuckle** [comment]
I see we already have explanatory text. Maybe it is too wordy and everyone skips it?

**Comment by Tkael** [comment]
I took a stab at this on PR #247. Closing as complete pending review.
